BlueprintUI/Sources/Layout/Flow.swift

```diff
@@ -116,6 +116,25 @@ extension Flow {
         case bottom
     }
 
+    /// When there is extra space in a run, how the extra space should be used.
```

Comment:
Since we may add a vertical version in the future, I'm calling it a "run" instead of a "row".